charts/k8s-monitoring/templates/agent_config/_cadvisor.river.txt

```diff
@@ -38,9 +38,10 @@ prometheus.scrape "cadvisor" {
 prometheus.relabel "cadvisor" {
   rule {
     source_labels = ["__name__"]
-    regex = "up|{{ join "|" .Values.metrics.cadvisor.allowList }}"
+    regex = "up|{{ join "|" (index .Values.metrics "cadvisor").allowList }}"
```

Comment:
Why change this? We were using the `index` method for `kube-state-metrics` and `node-exporter` because they had dashes in them. We don't need to make this change here.

Reply:
For consistency with the other files